Makefile.in

```diff
@@ -849,10 +850,10 @@ uninstall-am:
 
 
 copy-examples:
-	@echo "Copying examples folders..."
-	@if test "$(top_srcdir)" != "$(top_builddir)" ; then \
-		mkdir -p $(PWD)/examples ;\
-		rsync -a $(top_srcdir)/examples/* examples \
+	@if test $(EXAMPLES_DIR) != "$(top_srcdir)/examples" ; then \
+		echo "Copying examples folders into $(EXAMPLES_DIR)..." ; \
+		mkdir -p $(EXAMPLES_DIR) ; \
+		rsync -a $(top_srcdir)/examples/* $(EXAMPLES_DIR) \
 			--exclude decoupledibpm/oscillatingcylinder2dRe100_GPU ; \
```

Comment:
I guess we don't need to exclude this folder? Even though this folder is already copied before `make copy-examples`, copying it again won't hurt.

Reply:
Done.